jobs/ci-test-infra-bazel.sh

```diff
@@ -41,6 +41,6 @@ case "${rc}" in
     *) echo "Unknown exit code: ${rc}" ;;
 esac
 
-./images/pull_kubernetes_bazel/coalesce.py
+/coalesce.py
```

Comment:
I suspect that an absolute path reference like this is going to make this job harder to test locally (outside of the image). I don't know if that's a non-goal of this particular job, but in general, I thought we tried to have jobs that we could run using just `bootstrap.py` or even just directly, if you've set up your environment properly.

How do you feel about stripping the leading slash and just making sure that the script is in `$PATH` to find it? You'll likely have to add `/` to the `PATH` in the Dockerfile, but it seems like that would be more forgiving to local development.